config/prod/ecmonsen-emorypipeline-ci.yaml

```diff
@@ -0,0 +1,9 @@
+template_path: ecmonsen-emorypipeline-ci.yaml
+stack_name: {{ var.stackname | default("ecmonsen-emorypipeline-ci") }}
```

Comment:
AWS will create multiple stacks if names keep changing therefore this needs to be static.

Reply:
Done